docs/tutorials/losses_focalloss.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>It is good to add <code>compile()</code>  and <code>fit()</code>  in <code>Train and Evaluate</code>  and not in <code>Building The Model</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/1440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>